Sources/Build/BuildDescription/ProductBuildDescription.swift

```diff
@@ -367,6 +367,10 @@ public final class ProductBuildDescription: SPMBuildCore.ProductBuildDescription
 
         return flags
     }
+
+    func codeSigningArguments(plistPath: AbsolutePath, binaryPath: AbsolutePath) -> [String] {
+        ["codesign", "--force", "--sign", "-", "--entitlements", plistPath.pathString, binaryPath.pathString]
```

Comment:
It might be a good idea to have a command line option to let you specify the code signing identity (here we're using `-`).

Reply:
IMO that should come as a separate PR that would make this facility a bit more general.

Reply:
Sounds good. I only mention it because people who are aware of how this works will pretty quickly notice that we're signing things and ask how to sign with a different identity.

Reply:
That's fair, but signing with a different identity or with a different identity requires a public API design and likely an evolution proposal. Scoping this to one specific case means we don't have to block backtraces with this potential future design work.